test/functional/services/listing_table.ts

```diff
@@ -65,34 +65,6 @@ export function ListingTableProvider({ getService, getPageObjects }: FtrProvider
       await searchFilter.click();
     }
 
-    private async getAllItemsNamesOnCurrentPage(): Promise<string[]> {
```

Comment:
Agreed that both of these methods are unused and can be deleted. I also verified that all the _other_ ListingTableProvider methods are still used.